lib/file.js

```diff
@@ -22,9 +22,9 @@ module.exports = function(fileArray, source, options) {
   }
 
   var vinylFiles = fileArray.map(function(file) {
-    return new gutil.File({
+    return new Vinyl({
       cwd: "",
-      base: "",
+      base: undefined,
```

Comment:
I had prepared a PR and refactors to https://github.com/alexmingoia/gulp-file/pull/9
I do see that `cwd` & `base` is used in creating a new Vinyl. I did not keep the properties in thoughts that it was unneeded.

Reply:
OK, you're right that you can probably remove these properties. I just went for the minimum changes.
Thanks for merging it.